.github/workflows/ci.yml

```diff
@@ -548,6 +548,13 @@ jobs:
         with:
           workflow_id: ${{ env.WORKFLOW_ID }}
           access_token: ${{ github.token }}
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: "Validate OpenAPI spec"
+        run: |
+          pip install PyYAML
```

Comment:
@potiuk i added the install command here directly because it's only one line, let me know if you prefer me to wrap everything into one script inside `scripts/ci` instead.

Reply:
Should we add new linter at all? I think we can also add new rules to current linters 
https://www.github.com/wework/speccy/tree/master/rules

Reply:
https://www.github.com/wework/speccy/tree/master/rules%2Fdefault.yaml
Please look at: operation-operationId

Reply:
https://meta.stoplight.io/docs/spectral/README.md?_ga=2.94361181.1790267361.1595478554-125598717.1595478554

> What is the difference between Spectral and Speccy
Speccy was a great inspiration for Spectral, but was designed to work only with OpenAPI v3. Spectral can apply rules to any JSON/YAML object (including OpenAPI v2/v3 and AsyncAPI).

> Speccy has been abandoned, but Spectral is steaming ahead, adding loads of functionality like custom functions, exceptions, and AsyncAPI support.

Reply:
@mik-laj before I look into speccy, this is not something we are currently using right?

Reply:
https://github.com/apache/airflow/blob/master/.pre-commit-config.yaml#L194-L204
We already have two checks.

Reply:
ha, my bad, today i learned ripgrep doesn't search keyword in hidden files by default :( Thanks for pointing that out, i will switch to use precommit hook.

Reply:
@mik-laj switched to validate with speccy.